File: data/src/main/java/com/imadji/arch/data/entity/MovieData.java
package com.imadji.arch.data.entity;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by imadji on 06/21/2019.
 */

public class MovieData {

    @SerializedName("id")
    @Expose
    public long id;

    @SerializedName("title")
    @Expose
    public String title;

    @SerializedName("original_title")
    @Expose
    public String originalTitle;

    @SerializedName("overview")
    @Expose
    public String overview;

    @SerializedName("release_date")
    @Expose
    public String releaseDate;

    @SerializedName("vote_average")
    @Expose
    public double voteAverage;

    @SerializedName("poster_path")
    @Expose
    public String posterPath;

    @SerializedName("backdrop_path")
    @Expose
    public String backdropPath;
}

File: data/src/main/java/com/imadji/arch/data/entity/MovieData.kt
package com.imadji.arch.data.entity

import com.google.gson.annotations.SerializedName

data class MovieData(
        @SerializedName("id") val id: Long,
        @SerializedName("title") val title: String,
        @SerializedName("overview") val overview: String,
        @SerializedName("poster_path") val posterPath: String,
        @SerializedName("backdrop_path") val backdropPath: String,
        @SerializedName("release_date") val releaseDate: String,
        @SerializedName("vote_average") val voteAverage: Float,
        @SerializedName("original_title") val originalTitle: String,
        @SerializedName("original_language") val originalLanguage: String,
        @SerializedName("genre_ids") val genreIds: MutableList<Int>
)